pkg/config/config.go

```diff
@@ -192,6 +200,37 @@ func (c *Config) PrintConfig() error {
 	return nil
 }
 
+// ClearAllCredentials clears all API key fields from every Profile.
+func (c *Config) ClearAllCredentials() error {
+	runtimeViper := viper.GetViper()
+	var err error
+
+	for field, value := range runtimeViper.AllSettings() {
+		// TODO: ianjabour - ideally find a better way to identify projects in config
```

Comment:
Should we refactor some of the config code? @ob-stripe and I have felt like we've been needing too but haven't really yet...

There's a very basic function to check for this here: https://github.com/stripe/stripe-cli/blob/master/pkg/config/profile.go#L133-L136

Reply:
So the issue with using the function you linked above is that it assumes I have the profile on hand. In this case I need to find all profiles in the config. Being that we've been only working with one profile at a time up till now, I wasn't able to find anything that could identify a key in the config as a profile.